src/utils/IdealGraphVisualizer/ServerCompiler/src/main/resources/com/sun/hotspot/igv/servercompiler/filters/hideExceptionBlocks.filter

```diff
@@ -0,0 +1,18 @@
+// Hide exception blocks.
+
+var f = new RemoveBlockFilter("Hide exception blocks");
+f.addRule(
```

Comment:
A filter can consist of multiple rules, see e.g. `ServerCompiler/src/main/resources/com/sun/hotspot/igv/servercompiler/filters/structural.filter`, so I think it makes sense to pass them separately. What could be done to reduce this complexity is to define more helper methods in `Filter/src/main/resources/com/sun/hotspot/igv/filter/helper.js`. As you suggest, I think this would be best done for all filters at once in a separate RFE.